### ISSUE TITLE
Pretrained models

### ISSUE THREAD

**Issue by Zumbalamambo** [kohpai/Pedestrian-Detection#1]
Can you please share the pretrained model?

**Comment by kohpai** [comment]
Yes, sure. But this project is still in progress and, honestly, it's not ready to predict any images.  If you still want the pertained model, just tell me again, I will upload it. 😅

**Comment by kohpai** [comment]
Yes, sure. But this project is still in progress and, honestly, it's not ready to predict any images. If you still want the pertained model, just tell me again, I will upload it. 😅

**Comment by Zumbalamambo** [comment]
Ya please upload it... hehe and also provide a gist on how I can retrain it. let me open a pull request after training

**Comment by kohpai** [comment]
Sorry for my late reply. Now we're moving on to use pretrained models from [YOLO](https://pjreddie.com/darknet/yolo/) and trying to improve accuracy on pedestrian detection. This is our old weight files [pedestrian-32x80x1-weights](https://kohpai.github.io/files/neuralnetworks/pedestrian-32x80x1-weights): input size is 32x80 in gray-scale, [pedestrian-32x80x3-weights](https://kohpai.github.io/files/neuralnetworks/pedestrian-32x80x3-weights): input size is 32x80 in RGB, and [pedestrian-48x96x1-weights](https://kohpai.github.io/files/neuralnetworks/pedestrian-48x96x1-weights): input size is 48x96 in gray-scale.

The first 2 weight files were trained using NICTA dataset and the last file was trained using Daimler dataset. The accuracy on test dataset from NICTA was really good (almost 98%), but when applied with sliding window method, the accuracy dropped to around 50%. That's why we moved to new model. We used TensorFlow to train all weights but I have no experience on retrain model with TensorFlow so I may cannot help you with that. But I believe you can find it easily on the internet.

Thank you.
